docker/Makefile

```diff
@@ -1,6 +1,6 @@
 VERSION := 1.0.0
-ROOT_IMAGE ?= alpine:3.12
-CERT_IMAGE := alpine:3.12
+ROOT_IMAGE ?= alpine:3.13
+CERT_IMAGE := alpine:3.13
 GOLANG_IMAGE := golang:1.15-alpine
```

Comment:
not sure why we need go, but I would bump this to 1.16 as well

Reply:
Sure, I will bump it up. I think we need go for delve to work in debug images.